.github/CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,74 @@
+# Contributor Covenant Code of Conduct
```

Comment:
I agree to abide by this code of conduct.

Reply:
Are we all good to merge?

Reply:
Yeah. GitHub did something weird though, I think I have to make a duplicate review.